tmt/export.py

```diff
@@ -354,6 +350,27 @@ def export_to_nitrate(test):
         nitrate_case.identifier), fg='magenta'))
 
 
+def return_markdown_file():
+    """ Return path to the markdown file """
+    files = '\n'.join(os.listdir())
+    reg_exp = r'.+\.md'
```

Comment:
This regex will match files that do not end with `.md`. For example it matched vim temp file (*.md.swp). Not sure if it is intentional.

Reply:
Good catch @hegerj !
Updated by adding `$` at the end of regex.